src/odemis/driver/actuator.py

```diff
@@ -1404,7 +1404,9 @@ def __init__(self, name, role, children, caxes_map, axis_name, positions, fallba
                         positions. Fallback position can be equal to one of the positions allowed. If fallback is not
                         equal to one of the positions allowed, it is not possible to request moving to this fallback
                         position.
-        atol: absolute tolerance in the position of each child
+        atol (float, float): absolute tolerance in the position of each child
+        cycle (float or None): one complete turn of rotators until they reach position 0 again;
```

Comment:
`cycle (list of (float or None)): for each axis, the length of a full rotation until it reaches position 0 again`...